### DISCUSSION TITLE
自定义主页

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#4048] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/42478088/e52c0741-8b41-4f68-93ad-ce72d6e061a4)
。。。。

**Comment by RainbowTaro2022** [comment]
> ![image](https://private-user-images.githubusercontent.com/42478088/340058256-e52c0741-8b41-4f68-93ad-ce72d6e061a4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg1MDI2NzQsIm5iZiI6MTcxODUwMjM3NCwicGF0aCI6Ii80MjQ3ODA4OC8zNDAwNTgyNTYtZTUyYzA3NDEtOGI0MS00ZjY4LTkzYWQtY2U3MmQ2ZTA2MWE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE2VDAxNDYxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4N2VjNDkxYzRmNjM5NTdkNTdiYzY0MmJlNjUzOTQ1OGZhYWYwZThjMjkyMjI4MGU3OWE4MGZlZDkxYTVkOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yY-9yL-S7NiI2dFTAU2XH4NqKe1YP83aTI1S8xBbMgo) 。。。。

指的是每日整合包推荐的错误

**Comment by allMagicNB** [comment]
字面意思

**Comment by allMagicNB** [comment]
@wkea

**Comment by WForst-Breeze** [comment, marked as answer]
第三方主页请前往反馈地址 https://sodamc.com/31402-29042412.html 或联系作者 @wkea
